Source/Scene/Cesium3DTileset.js

```diff
@@ -528,20 +484,20 @@ define([
 // TODO: same TODO as above.
                     }
 
-                    if (!allChildrenLoaded) {
+                    if (!t.isRefinable()) {
                         // Tile does not meet SSE.  Add its commands since it is the best we have and request its children.
-                        if (!t.hasTilesetContent) {
-                            selectTile(selectedTiles, t, fullyVisible, frameState);
-                        } else {
-                            selectTileWithTilesetContent(tiles3D, selectedTiles, t, fullyVisible, frameState, additiveRefinment);
-                        }
+                        selectTile(selectedTiles, t, fullyVisible, frameState);
 
                         if (outOfCore) {
                             for (k = 0; (k < childrenLength) && requestScheduler.hasAvailableRequests(); ++k) {
                                 child = children[k];
 // TODO: we could spin a bit less CPU here and probably above by keeping separate lists for unloaded/ready children.
                                 if (child.isContentUnloaded()) {
-                                    requestContent(tiles3D, child);
+                                    requestContent(tiles3D, child, outOfCore);
+                                } else if (!child.isRefinable()) {
+                                    // Child needs to load its children in order to become refinable
```

Comment:
Can we expand on this comment a bit, e.g., "needs to load its children or, in the case of empty child tiles used to accelerate culling, its ancestors with content need to be loaded"